nebula/ui/components/MZCheckBox.qml

```diff
@@ -71,7 +71,7 @@ CheckBox {
 
             PropertyChanges {
                 target: checkBoxIndicator
-                border.color: checkBox.checked || checkBox.activeFocus ? MZTheme.theme.blueHovered : MZTheme.theme.fontColorDark
+                border.color: if(checBox.enabled) (checkBox.checked || checkBox.activeFocus ? MZTheme.theme.blueHovered : MZTheme.theme.fontColorDark)
```

Comment:
typo in `if (checkBox.enabled)` and then`checkBox.checked || checkBox.activeFocus ? MZTheme.theme.blueHovered : MZTheme.theme.fontColorDark` should be wrapped in curly brackets instead of parenthesis.

Reply:
fixed the typo but left off the brackets to look consistent with the other property changes since its still a one-liner